src/lt/plugins/clojure.cljs

```diff
@@ -68,13 +68,15 @@
           (run-local-server (clients/client! :nrepl.client))))))
 
 (behavior ::on-eval.clj
+          :desc "Clojure: Eval editor content"
           :triggers #{:eval}
           :reaction (fn [editor]
                       (object/raise clj-lang :eval! {:origin editor
                                                      :info (assoc (@editor :info)
                                                              :print-length (object/raise-reduce editor :clojure.print-length+ nil)
                                                              :code (watches/watched-range editor nil nil nil))})))
 (behavior ::on-eval.cljs
+          :desc "Clojurescript: Eval editor content"
```

Comment:
I have opened https://github.com/LightTable/LightTable/pull/2296 to document this.